src/lightning_app/api/__init__.py

```diff
@@ -1,3 +1,15 @@
-from lightning_app.api.http_methods import Delete, Get, Post, Put
+from lightning_app.api.http_methods import Delete, Get, HttpMethod, Post, Put
+from lightning_app.api.request_types import APIRequest, BaseRequest, CommandRequest, DeltaRequest, RequestResponse
 
-__all__ = ["Delete", "Get", "Post", "Put"]
+__all__ = [
+    "HttpMethod",
+    "Delete",
+    "Get",
```

Comment:
I wanted to expose only `"Delete", "Get", "Post", "Put"`. I think everything else can be moved private.